integration_tests/src/main/python/cache_test.py

```diff
@@ -285,3 +285,67 @@ def helper(spark):
         return df.selectExpr("a")
 
     assert_gpu_and_cpu_are_equal_collect(helper)
+
+
+# No need to run these tests without rapids cache serializer since the 'InMemoryTableScanExec'
+# node will fall back to CPU for such case. Then there is no GPU things joined in for the
+# cache/uncache operations.
+@pytest.mark.skipif(is_before_spark_330() or not is_with_rapids_cache_serializer(),
```

Comment:
Why the extra check "is_with_rapids_cache_serializer()"? Does the test fail with the default serializer?

Reply:
Thanks for review.
No, it passes. As said in the comment, with the default serializer, `InMemoryTableScanExec` will run on CPU, then no GPU things here for cache. So do we still need to test ?

Reply:
So, the original intent for cache_test.py was not to test PCBS or GpuInMemoryTableScanExec. It's testing if output from GPU exec's can successfully be cached.

Reply:
Good to know, thanks.

Reply:
I removed the serializer check